firmware/application/apps/ui_fileman.cpp

```diff
@@ -150,7 +154,8 @@ void FileManBaseView::load_directory_contents(const fs::path& dir_path) {
             continue;
 
         if (fs::is_regular_file(entry.status())) {
-            if (!filtering || path_iequal(entry.path().extension(), extension_filter))
+            if (!filtering || path_iequal(entry.path().extension(), extension_filter) ||
```

Comment:
Minor: This feels bad to me. Don't have to fix for now, but ideally the filesystem's directory_iterator should handle this or allow a vector of allowed extension to be passed in so we can more easily support an arbitrary list of extensions.

Reply:
I cleaned this up slightly by adding functions to file.cpp (there might be a better place to put them in future PR)